test/e2e/framework/util.go

```diff
@@ -143,9 +142,6 @@ const (
 
 	// TODO(justinsb): Avoid hardcoding this.
 	awsMasterIP = "172.20.0.9"
-
-	// ssh port
-	sshPort = "22"
```

Comment:
how about don't move or move it to ssh subpackage.

Reply:
There are so many port in test/e2e , we can use a  common  package in e2e core framework in the future.

Reply:
That requires changes more files like e2e/network/service.go, e2e/storage/flexvolume.go, e2e/framework/ssh/ssh.go and e2e/framework/node/resource.go
I don't want to include such changes into this PR for keeping the scope simple and clear.

Reply:
sure. This   subpackage  kubesystem  is good for me. 
But sshPort seems to be closer to the ssh subpackage. 
It is not easy to reuse this constant here. How about adding a comment ? Here is just a suggestion. 😄

Reply:
I mean that  ``` const sshPort  ``` move to ssh  package.

Reply:
I guess we will do that soon within different PR even if not doing that ;-)

Reply:
@tanjunchen Do you have still a concern about this PR?

Reply:
@oomichi 
no concern. Thanks
/lgtm